.travis.yml

```
@@ -21,7 +21,22 @@ notifications:
 after_success:
   - julia -e 'using Pkg; Pkg.add("Coverage"); Pkg.dir("FFTW"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
 
+branches:
+  only:
+  - master
+  - gh-pages # For building documentation
+  - /^testing-.*$/ # testing branches
+  - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
+
+cache:
+  directories:
+    - $HOME/.julia/artifacts
+
 jobs:
+  exclude:
+    - os: freebsd
+      env:
+        - JULIA_FFTW_PROVIDER=MKL
```

```suggestion
      env: JULIA_FFTW_PROVIDER=MKL
```
? Not sure it will make a difference but worth a try I guess

---

:open_mouth: